cypress/e2e/shared/variables.test.ts

```diff
@@ -18,7 +18,7 @@ describe('Variables', () => {
   it('can CRUD a CSV, upload, map, and query variable and search for variables based on names', () => {
     // Navigate away from and back to variables index using the nav bar
     cy.getByTestID('nav-item-dashboards').click()
-    cy.getByTestID('nav-item-settings').click()
+    cy.getByTestID('nav-item-settings').click({force: true})
```

Comment:
New navigation when it is collapsed have an overlaying button for the submenu. Force true was added to go around that.